File: spring/admin/src/main/java/com/learnwy/admin/user/repository/UserRepository.java
package com.learnwy.admin.user.repository;

import com.learnwy.admin.user.model.User;
import org.springframework.data.repository.CrudRepository;

/**
 * date 2017/8/25
 * create by learnwy
 * lis is GPL-3.0
 */
public interface UserRepository extends CrudRepository<User, Long> {

}

File: kotlin/_system/src/main/kotlin/com/learnwy/system/repositories/UserRepository.kt
package com.learnwy.system.repositories

import com.learnwy.system.dto.User
import org.springframework.data.domain.Page
import org.springframework.data.domain.Pageable
import org.springframework.data.repository.CrudRepository

interface UserRepository : CrudRepository<User, Long> {
    fun findAll(pageRequest: Pageable): Page<User>;
}